docs/components/concepts/expressions.md

```diff
@@ -180,7 +180,14 @@ More functions for string values are available as [built-in string functions](/c
 
 ### Temporal expressions
 
-The current date and date-time can be accessed using the built-in functions `today()` and `now()`. To store the current date or date-time in a variable, convert it to a string using the built-in function `string()`.
+The current date and date-time can be accessed using the built-in functions `today()` and `now()`.
+Storing a temporal value as a variable will store it in the ISO 8601 format.
```

Comment:
> store it in the ISO 8601 format.

❓ Is it clear to users that it will be stored as a string? Perhaps it is, but we could also choose to document it explicitly. WDYT?

> store it in the ISO 8601 format.

❌ Actually, this concerns FEEL's extension of the ISO-8601 representation that adds support for a Zone ID. The ISO-8601 only supports the UTC/Z and time offsets. We should clarify this. See the snippet from DMN 1.3 Spec (10.3.4.1) below. You can also look at 10.3.2.3.4, 10.3.2.3.6 for more details about FEEL's usage of the UTC/Z and time offsets.

> • **time string** – either a string value in the lexical space of the time datatype specified by XML Schema Part 2 Datatypes; or a string value that is the extended form of a local time representation as specified by ISO 8601, followed by the character "@", followed by a string value that is a time zone identifier in the IANA Time Zones Database (http://www.iana.org/time-zones)
• **date time string** – a string value consisting of a date string value, as specified above, optionally followed by the character "T" followed by a time string value as specified above.

Reply:
> Actually, this concerns FEEL's extension of the ISO-8601 representation that adds support for a Zone ID. The ISO-8601 only supports the UTC/Z and time offsets. We should clarify this

How do you propose I write this down? Looking at the snippet there is no easy word to use here. I can write `store it in an extended form of a local time representation as specified by ISO 8601` but I feel like that's vague and confusing.

Reply:
@korthout I'm actually not convinced what you say here it correct. If I read [this issue](https://github.com/camunda/zeebe/issues/10307) it contains some examples which seem to verify that it's stored in an ISO 8601 format.

Reply:
Turned it into:
`Storing a temporal value as a variable will store it as a timezone extended ISO 8601 formatted string.`

Reply:
Perhaps we should write that we store it in the FEEL format (an extended ISO 8601 format, see https://docs.camunda.io/docs/next/components/modeler/feel/language-guide/feel-data-types/#time for details)